paper/paper.md

```diff
@@ -78,7 +78,7 @@ Custom gradients can be added to these functions.
 
 Automatic differentiation is a core feature of JAX and often one of the main reasons for using it.
 Thus, it is essential that custom functions registered with JAX support automatic differentiation.
-In the following, we will outline which functions our package respectively JAX requires to enable automatic differentiation.
```

Comment:
I think what's meant is "... which functions our package requires to enable automatic differentiation via JAX".

Reply:
Sounds good to me!